Makefile

```diff
@@ -0,0 +1,17 @@
+DOCKER_IMAGE = resinos-tests
+
+test: env.list build-docker-image
```

Comment:
We are pointing to `env.list`, however that file doesn't exist in the root of the project. Is the user supposed to create it? If so, can we document it in the README?

Reply:
The user will have to create it after a template and modify it as requested and will be documented in README.